illuminatio-runner.dockerfile

```diff
@@ -40,4 +40,4 @@ COPY src /src/app/src
 
 USER runner
```

Comment:
use nurmeric user here, too? Requires adding `-u 1000` in line 19

Reply:
The user is never used so I wont change it here